lib/workers/repository/onboarding/common.ts

```diff
@@ -0,0 +1,30 @@
+import hasha from 'hasha';
+import { configFileNames } from '../../../config/app-strings';
+import type { RenovateConfig } from '../../../config/types';
+import { logger } from '../../../logger';
+
+export const defaultConfigFile = (config: RenovateConfig): string =>
+  configFileNames.includes(config.onboardingConfigFileName!)
+    ? config.onboardingConfigFileName!
+    : configFileNames[0];
+
+export class OnboardingState {
```

Comment:
Although the getter setters here does not hold any logic and a regular variable could suffice, the added benefit of logging on every interaction with it is enough in my view for it to exist.